src/proto/finetuning.proto

```diff
@@ -0,0 +1,105 @@
+syntax = "proto3";
+package gooseai;
+
+option go_package = "github.com/stability-ai/api-interfaces/gooseai/finetuning";
+
+enum FineTuningMode {
+  FINE_TUNING_MODE_UNSPECIFIED = 0; // No mode specified
+  FINE_TUNING_MODE_FACE = 1;        // Fine tuning a face model
+  FINE_TUNING_MODE_STYLE = 2;       // Fine tuning a style model
+  FINE_TUNING_MODE_OBJECT = 3;      // Fine tuning an object model
+}
+
+enum FineTuningStatus {
+  FINE_TUNING_STATUS_NOT_STARTED = 0; // Model not yet started
+  FINE_TUNING_STATUS_RUNNING = 1;     // Model is currently running
+  FINE_TUNING_STATUS_COMPLETED = 2;   // Model has completed successfully
+  FINE_TUNING_STATUS_FAILED = 3;      // Model has failed
+  FINE_TUNING_STATUS_SUBMITTED = 4;   // Model has been submitted
+}
+
+message FineTuningModel {
+  string id = 1;                    // ID of the model, UUIDv4
+  string user_id = 2;               // ID of user who created the model
+  string name = 3;                  // a readable model name
+  optional FineTuningMode mode = 4; // the mode of the model
+  optional string object_name = 5;  // freeform text description of object, should only be set when mode is OBJECT
+  string project_id = 6;            // project ID with assets to be used for fine tuning
+  double duration = 7;              // how long the model took to fine tune
```

Comment:
What's the unit here?

Reply:
Nice catch, seconds I think. 

Would you like the format to be duration_seconds?  Or just have a comment indicating the unit?

Reply:
Something like `duration_secs` would be my top choice.  But a comment would also be fine since it'll end up in generated outputs/docstrings.

Reply:
I've indicated the unit in the comment now.